lifelines/statistics.py

```diff
@@ -46,14 +45,14 @@ def logrank_test(event_times_A, event_times_B, event_observed_A=None, event_obse
         event_observed_B = np.ones(event_times_B.shape[0])
 
     event_times = np.r_[event_times_A, event_times_B]
-    groups = np.r_[np.zeros(event_times_A.shape[0]), np.ones(event_times_B.shape[0])]
+    groups = np.r_[np.zeros(event_times_A.shape[0], dtype=int), np.ones(event_times_B.shape[0], dtype=int)]
```

Comment:
this was cosmetic. instead of column names being `group:1.0`, etc., I prefer `group:1`.